### DISCUSSION TITLE
Send data to parent layout

### DISCUSSION THREAD

**Discussion by digitaltoad** [remix-run/remix#4033] [Q&A] [answered]
This may have been asked or answered previously but I can't seem to find anything about it. I want to update a title that is in a parent layout, possibly multiple levels deep. What is the Remix way to do such a thing. As a simple example:

I navigate to `/posts` and the title on the parent layout should be `Posts`. If I navigate to a single post at `/posts/1`, the parent layout should have a title matching the post title. I thought about using the meta export but I'm not sure how to access the data from that in the parent layout.

I know in a perfect world, I could just rework the layout to colocate the title within the view that cares. I also know I can use context and just update the layout after render. Is there a way that is preferable in Remix though?

**Comment by sergiodxa** [comment, marked as answer]
You can use the useMatches hook on the parent route to access the nested routes loader data and handle export.

**Reply by digitaltoad** [reply]
Ah neat! I honestly thought that in using this method the layout would have to know more about the child route than it does. Works great. Thank you!
